tests/test_encrypted_fields.py

```diff
@@ -4,15 +4,19 @@
 
 import pytest
 from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
 from django.db import connection, models
+from django.forms.widgets import Textarea, TextInput
 from django.test import TestCase
+from django.test.utils import override_settings
 
 from .testapp.models import Secret
 
 # Only perform encrypted fields tests if keyczar is present. Resolves
 # http://github.com/django-extensions/django-extensions/issues/#issue/17
 try:
-    from django_extensions.db.fields.encrypted import EncryptedTextField, EncryptedCharField  # NOQA
+    from django_extensions.db.fields.encrypted import BaseEncryptedField, \
+        EncryptedTextField, EncryptedCharField
```

Comment:
Can we either put all the imports on one line or make multiple import lines instead of using a `\` ?

Reply:
I've put imports in one line. What style of multiple lines would you prefrer (I will kept that in my memory for the furure) ?

Reply:
I actually prefer to just use a single line until it becomes too unwieldy.
This one is still just 108 chars wide.. so good enough for me to keep on one line...

Personally I would break pretty late ... maybe around 200 chars or so.. 

What I would actually prefer but isort does not support is breaking like this:
```
from third_party import lib1, lib2, lib3
from third_party import lib4, lib5
```

( I have added `line_length = 128` to isort config in setup.cfg. )

For now I would just keep this import on one line.

Reply:
ok, its in one line now.